generator/whitelist.ts

```diff
@@ -286,6 +286,11 @@ const whitelist: WhitelistConfig[] = [
         basePath: 'relay/resource-manager',
         namespace: 'Microsoft.Relay',
     },
+    {
+        basePath: 'recoveryservicessiterecovery/resource-manager',
+        namespace: 'Microsoft.RecoveryServices',
+        suffix: 'SiteRecovery',
```

Comment:
I've added this suffix because there are other Microsoft.SiteRecovery folders in the specs repo, and we want to avoid a clash. Please could you re-run autogeneration and commit the changes?